### PR TITLE
Ndelangen/tags in autocomplete value

### PR DESCRIPTION
**Pull request by ndelangen**
* CHANGED the `keyword` property in info.json files to `tag`.
* CORRECTED documentation regarding this property.
* IMPROVED tag info for `docs/info-json/info.json`. to make it easier to find, when searching docs for a what a property within an `info.json` does.

### CODE REVIEW

**Comment by operatino** [comment]
Hi,

Sorry for taking it so long, I'm currently very busy with deadlines and new home renovation. I'll try to check it today evening or tomorrow during the day.

File: assets/js/modules/search.js [hunk 1]
```diff
@@ -72,18 +72,18 @@ Search.prototype.prepareAutoCompleteData = function() {
...
                 ? pagesData[page]['name']
                 : ""; //get cat name
-            var prepareKeywords = '';
+            var tags = '';
             var autocompleteValue = targetPage.title;
 
-            if (keywords && keywords !== '') {
-                prepareKeywords += ', ' + keywords;
+            if (tag && tag.length) {
+                tags += ', ' + tag.join(' ');
```
Comment (operatino) [inline assets/js/modules/search.js]: Please us `.join(',')`, so the final `autocompleteValue` will have nicely formatted string with columns after each word like `pageName, tag1, tag2`. Now you're generating `pageName, tag1 tag2 tag3`.

**Comment by operatino** [comment]
Good job, I have only one comment about end format for autocomplete value. After it will be resolved, I can merge the changes.

**Comment by operatino** [comment]
Thank for the PR, it now look good. Sorry it took so long after last comment, I was expecting some email from GitHub but it didn't came :(

**Comment by ndelangen** [comment]
Really glad to help, and have my first PR accepted. Happy day for me :smile:

**Comment by operatino** [comment]
@ndelangen Congrats with the first PR then!

**Comment by operatino** [comment]
@ndelangen Btw, it seems like your office is 7 min walk from my :) We can have a lunch some day if you like. I'm working at Liberty Global at Schiphol Rijk.

**Comment by ndelangen** [comment]
How about next Wednesday? I'll be @ our HQ then. Shall I come to your office?

**Comment by operatino** [comment]
@ndelangen yeah, let's do it Wednesday. Please send me your phone number on r@rhr.me.

We have here a really nice, cheap cafeteria and cool lounge spaces, but if you like, I can come to your place.

**Pull request merged**